### ISSUE TITLE
Rayman 3: Hoodlum Havoc. Movie plays. Crashes before menu.

### ISSUE THREAD

**Issue by PatrickvL** [Cxbx-Reloaded/game-compatibility#126]
_From @ObiKKa on April 21, 2017 1:19_

The Russian tester on his Youtube channel "Emulators of game console" has uploaded a video test of this game Rayman 3 on April 17, 2017 with the CXBX Reloaded (April14.2017) build.

https://www.youtube.com/watch?v=W3XKLz4FzWI

* Description:
"My Spec:
CPU: Intel i5-3570K@3.6 GHz (MB MSI B75A-G43),
GPU: Nvidia GTX 1050ti 4GB (MSI Gaming X),
RAM 8GB DDR3-1600 MHz (Kingston HyperX),
Windows 10 x64 Pro (ForceWare 378.66)
--
Rayman 3: Hoodlum Havoc (Xbox)
Status: Loadable / Intro
Realtime Footage"

_Copied from original issue: Cxbx-Reloaded/Cxbx-Reloaded#386_

**Comment by sonicjam717** [comment]
I'm getting a Read Disc error in the game.

![capture](https://user-images.githubusercontent.com/30691915/35630798-49367fd2-0670-11e8-8344-8055025d62b1.PNG)

My Specs:
CPU: AMD FX8150 3.6ghz, boost to 4.2ghz
GPU: AMD R9 290 4gb, driver Crimson 18.1.1
RAM: 8gb ddr3
Windows 10 Pro 64bit

**Comment by CakeLancelot** [comment]
> I'm getting a Read Disc error in the game.

Using non-Xbox specific tools to extract .iso files (such as winRAR or 7zip) is known to cause this error, can you try again - this time using a tool such as Qwix or C-Xbox tool to extract?

**Comment by sonicjam717** [comment]
Yeah probably I had this game stored on my xbox and works fine on it. I'm checking to see if that was the issue.

**Comment by sonicjam717** [comment]
Latest build Feb 9 2018,
Game goes pass the first loading screen:
![capture](https://user-images.githubusercontent.com/30691915/36057023-99059dc2-0dd7-11e8-8897-549d5c091750.PNG)

[KrnlDebug.txt](https://github.com/Cxbx-Reloaded/game-compatibility/files/1712874/KrnlDebug.txt)

My Specs:
CPU: AMD FX8150 3.6ghz, boost to 4.2ghz
GPU: AMD R9 290 4gb, driver Crimson 18.1.1
RAM: 8gb ddr3
Windows 10 Pro 64bit

**Comment by gandalfthewhite19890404** [comment]
Game goes ingame now, but with graphic issues:
![image](https://user-images.githubusercontent.com/24764335/71303477-ba10f800-23d2-11ea-9d6d-9dc428842e9c.png)

**Comment by FerPozzetti** [comment]
No issues found.

Cxbx-Reloaded Version f1d09af (Jun  8 2020)

![1](https://user-images.githubusercontent.com/56910468/84849519-dc0ad080-b02b-11ea-94d0-d641d0fc09a6.jpg)
![2](https://user-images.githubusercontent.com/56910468/84849525-de6d2a80-b02b-11ea-97c4-fdc98d2c4e11.jpg)

[CxbxDebug.txt](https://github.com/Cxbx-Reloaded/game-compatibility/files/4790095/CxbxDebug.txt)
[Xbe.txt](https://github.com/Cxbx-Reloaded/game-compatibility/files/4790097/Xbe.txt)
[KrnlDebug.txt](https://github.com/Cxbx-Reloaded/game-compatibility/files/4790098/KrnlDebug.txt)

**Comment by Zankslim** [comment]
Works to 100% but missing audio/Sound over half of the game.

**Comment by Margen67** [comment]
@Zankslim Please provide KrnlDebug.txt.

**Comment by RibShark** [comment]
Also missing audio from the majority of the game here.
